test/extended/util/image/image.go

```diff
@@ -47,6 +47,9 @@ func init() {
 		"docker.io/library/nginx:1.14-alpine":     23,
 		"docker.io/library/nginx:1.15-alpine":     24,
 		"docker.io/library/redis:5.0.5-alpine":    31,
+
+		// Used for stressing node
+		"k8s.gcr.io/stress:v1": -1,
```

Comment:
instead of this, let’s just add stress to the tools image: https://github.com/openshift/oc/blob/master/images/tools/Dockerfile

stress is packaged for rhel, right?

Reply:
@ravisantoshgudimetla mind updating, I totally agree with Clayton on this one :-)

Reply:
Sure.